From 96a275dbffcc967cdbd4c29c9a3196a9518bae00 Mon Sep 17 00:00:00 2001
From: Igor Ivanov <igor.ivanov.va@gmail.com>
Date: Fri, 19 Jun 2020 16:29:25 +0300
Subject: [PATCH] Update systemctl files

Signed-off-by: Igor Ivanov <igor.ivanov.va@gmail.com>
---
 contrib/scripts/vma.service.in | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/contrib/scripts/vma.service.in b/contrib/scripts/vma.service.in
index e268173f..13457b2f 100644
--- a/contrib/scripts/vma.service.in
+++ b/contrib/scripts/vma.service.in
@@ -1,14 +1,12 @@
 [Unit]
-Description=VMA Daemon. Version: @VERSION@-@VMA_LIBRARY_RELEASE@
-After=network.target syslog.target
-Requires=network.target
+Description=VMA Daemon
+After=network.target
 
 [Service]
 Type=forking
 Restart=on-failure
 ExecStart=@prefix@/sbin/vma start
 ExecStop=@prefix@/sbin/vma stop
-ExecReload=@prefix@/sbin/vma restart
 RestartForceExitStatus=1 SIGTERM
 
 [Install]
-- 
2.18.1

